functions/Get-DbaDatabase.ps1

```diff
@@ -1,12 +1,15 @@
 ﻿function Get-DbaDatabase {
-	<#
+    <#
 		.SYNOPSIS
 			Gets SQL Database information for each database that is present in the target instance(s) of SQL Server.
 
 		.DESCRIPTION
 			The Get-DbaDatabase command gets SQL database information for each database that is present in the target instance(s) of
 			SQL Server. If the name of the database is provided, the command will return only the specific database information.
 
+			We also added a number of useful columns by default such as the Last used read and write Times for the Database using 
```

Comment:
That's what it was - Chrissy changed it - so whichever way is decided we can do :-)